Binary files ../original-apks/HtcSoundRecorder.apk/res/drawable-ldpi/eq_0.png and HtcSoundRecorder.apk/res/drawable-ldpi/eq_0.png differ
Binary files ../original-apks/HtcSoundRecorder.apk/res/drawable-ldpi/eq_1_light.png and HtcSoundRecorder.apk/res/drawable-ldpi/eq_1_light.png differ
Binary files ../original-apks/HtcSoundRecorder.apk/res/drawable-ldpi/eq_1.png and HtcSoundRecorder.apk/res/drawable-ldpi/eq_1.png differ
Binary files ../original-apks/HtcSoundRecorder.apk/res/drawable-ldpi/screen_bg.png and HtcSoundRecorder.apk/res/drawable-ldpi/screen_bg.png differ
Binary files ../original-apks/HtcSoundRecorder.apk/res/drawable-ldpi/screen_on.png and HtcSoundRecorder.apk/res/drawable-ldpi/screen_on.png differ
Binary files ../original-apks/HtcSoundRecorder.apk/res/drawable-ldpi/screen_playback.png and HtcSoundRecorder.apk/res/drawable-ldpi/screen_playback.png differ
Binary files ../original-apks/HtcSoundRecorder.apk/res/drawable-ldpi/timer_on_colon.png and HtcSoundRecorder.apk/res/drawable-ldpi/timer_on_colon.png differ
Binary files ../original-apks/HtcSoundRecorder.apk/res/drawable-ldpi/timer_on_no0.png and HtcSoundRecorder.apk/res/drawable-ldpi/timer_on_no0.png differ
Binary files ../original-apks/HtcSoundRecorder.apk/res/drawable-ldpi/timer_on_no1.png and HtcSoundRecorder.apk/res/drawable-ldpi/timer_on_no1.png differ
Binary files ../original-apks/HtcSoundRecorder.apk/res/drawable-ldpi/timer_on_no2.png and HtcSoundRecorder.apk/res/drawable-ldpi/timer_on_no2.png differ
Binary files ../original-apks/HtcSoundRecorder.apk/res/drawable-ldpi/timer_on_no3.png and HtcSoundRecorder.apk/res/drawable-ldpi/timer_on_no3.png differ
Binary files ../original-apks/HtcSoundRecorder.apk/res/drawable-ldpi/timer_on_no4.png and HtcSoundRecorder.apk/res/drawable-ldpi/timer_on_no4.png differ
Binary files ../original-apks/HtcSoundRecorder.apk/res/drawable-ldpi/timer_on_no5.png and HtcSoundRecorder.apk/res/drawable-ldpi/timer_on_no5.png differ
Binary files ../original-apks/HtcSoundRecorder.apk/res/drawable-ldpi/timer_on_no6.png and HtcSoundRecorder.apk/res/drawable-ldpi/timer_on_no6.png differ
Binary files ../original-apks/HtcSoundRecorder.apk/res/drawable-ldpi/timer_on_no7.png and HtcSoundRecorder.apk/res/drawable-ldpi/timer_on_no7.png differ
Binary files ../original-apks/HtcSoundRecorder.apk/res/drawable-ldpi/timer_on_no8.png and HtcSoundRecorder.apk/res/drawable-ldpi/timer_on_no8.png differ
Binary files ../original-apks/HtcSoundRecorder.apk/res/drawable-ldpi/timer_on_no9.png and HtcSoundRecorder.apk/res/drawable-ldpi/timer_on_no9.png differ
Binary files ../original-apks/HtcSoundRecorder.apk/res/drawable-ldpi/voice_recorder.png and HtcSoundRecorder.apk/res/drawable-ldpi/voice_recorder.png differ
Binary files ../original-apks/HtcSoundRecorder.apk/res/drawable-ldpi/voicerecorder_progress_button.png and HtcSoundRecorder.apk/res/drawable-ldpi/voicerecorder_progress_button.png differ
Binary files ../original-apks/HtcSoundRecorder.apk/res/drawable-mdpi/eq_0.png and HtcSoundRecorder.apk/res/drawable-mdpi/eq_0.png differ
Binary files ../original-apks/HtcSoundRecorder.apk/res/drawable-mdpi/eq_1_light.png and HtcSoundRecorder.apk/res/drawable-mdpi/eq_1_light.png differ
Binary files ../original-apks/HtcSoundRecorder.apk/res/drawable-mdpi/eq_1.png and HtcSoundRecorder.apk/res/drawable-mdpi/eq_1.png differ
Binary files ../original-apks/HtcSoundRecorder.apk/res/drawable-mdpi/screen_bg.png and HtcSoundRecorder.apk/res/drawable-mdpi/screen_bg.png differ
Binary files ../original-apks/HtcSoundRecorder.apk/res/drawable-mdpi/screen_on.png and HtcSoundRecorder.apk/res/drawable-mdpi/screen_on.png differ
Binary files ../original-apks/HtcSoundRecorder.apk/res/drawable-mdpi/screen_playback.png and HtcSoundRecorder.apk/res/drawable-mdpi/screen_playback.png differ
Binary files ../original-apks/HtcSoundRecorder.apk/res/drawable-mdpi/timer_on_colon.png and HtcSoundRecorder.apk/res/drawable-mdpi/timer_on_colon.png differ
Binary files ../original-apks/HtcSoundRecorder.apk/res/drawable-mdpi/timer_on_no0.png and HtcSoundRecorder.apk/res/drawable-mdpi/timer_on_no0.png differ
Binary files ../original-apks/HtcSoundRecorder.apk/res/drawable-mdpi/timer_on_no1.png and HtcSoundRecorder.apk/res/drawable-mdpi/timer_on_no1.png differ
Binary files ../original-apks/HtcSoundRecorder.apk/res/drawable-mdpi/timer_on_no2.png and HtcSoundRecorder.apk/res/drawable-mdpi/timer_on_no2.png differ
Binary files ../original-apks/HtcSoundRecorder.apk/res/drawable-mdpi/timer_on_no3.png and HtcSoundRecorder.apk/res/drawable-mdpi/timer_on_no3.png differ
Binary files ../original-apks/HtcSoundRecorder.apk/res/drawable-mdpi/timer_on_no4.png and HtcSoundRecorder.apk/res/drawable-mdpi/timer_on_no4.png differ
Binary files ../original-apks/HtcSoundRecorder.apk/res/drawable-mdpi/timer_on_no5.png and HtcSoundRecorder.apk/res/drawable-mdpi/timer_on_no5.png differ
Binary files ../original-apks/HtcSoundRecorder.apk/res/drawable-mdpi/timer_on_no6.png and HtcSoundRecorder.apk/res/drawable-mdpi/timer_on_no6.png differ
Binary files ../original-apks/HtcSoundRecorder.apk/res/drawable-mdpi/timer_on_no7.png and HtcSoundRecorder.apk/res/drawable-mdpi/timer_on_no7.png differ
Binary files ../original-apks/HtcSoundRecorder.apk/res/drawable-mdpi/timer_on_no8.png and HtcSoundRecorder.apk/res/drawable-mdpi/timer_on_no8.png differ
Binary files ../original-apks/HtcSoundRecorder.apk/res/drawable-mdpi/timer_on_no9.png and HtcSoundRecorder.apk/res/drawable-mdpi/timer_on_no9.png differ
Binary files ../original-apks/HtcSoundRecorder.apk/res/drawable-mdpi/voice_recorder.png and HtcSoundRecorder.apk/res/drawable-mdpi/voice_recorder.png differ
Binary files ../original-apks/HtcSoundRecorder.apk/res/drawable-mdpi/voicerecorder_progress_button.png and HtcSoundRecorder.apk/res/drawable-mdpi/voicerecorder_progress_button.png differ
diff -rupN ../original-apks/HtcSoundRecorder.apk/res/layout-mdpi/effect_panel.xml HtcSoundRecorder.apk/res/layout-mdpi/effect_panel.xml
--- ../original-apks/HtcSoundRecorder.apk/res/layout-mdpi/effect_panel.xml	2012-05-26 10:18:01.000000000 +0100
+++ HtcSoundRecorder.apk/res/layout-mdpi/effect_panel.xml	2012-05-26 10:08:53.000000000 +0100
@@ -1,27 +1,25 @@
 <?xml version="1.0" encoding="utf-8"?>
 <merge
   xmlns:android="http://schemas.android.com/apk/res/android">
-    <RelativeLayout android:orientation="horizontal" android:id="@id/effectViewLayout0" android:layout_width="wrap_content" android:layout_height="wrap_content" android:layout_marginLeft="@dimen/htc_man_effectViewlayout0_layout_margin_left" android:layout_marginTop="@dimen/htc_man_effectViewlayout0_layout_margin_top">
-        <RelativeLayout android:orientation="horizontal" android:id="@id/effectViewLayoutBackground" android:background="@drawable/screen_on" android:layout_width="wrap_content" android:layout_height="wrap_content" />
-        <RelativeLayout android:id="@id/effectViewLayout" android:layout_width="@dimen/htc_man_effectViewlayout_layout_width" android:layout_height="@dimen/htc_man_effectViewlayout_layout_height">
-            <RelativeLayout android:orientation="horizontal" android:id="@id/stateMessage2Layout" android:layout_width="fill_parent" android:layout_height="wrap_content" android:layout_marginTop="@dimen/htc_man_state_messag2layout_layout_margin_top" android:layout_marginRight="@dimen/htc_man_state_message2layout_layout_margin_right" android:baselineAligned="false" android:layout_centerHorizontal="true" style="@style/voiceRecordTextStytleB">
-                <TextView android:layout_gravity="left|center" android:id="@id/stateMessage2" android:layout_width="@dimen/htc_man_stateMessage2_layout_width" android:layout_height="wrap_content" android:layout_marginLeft="@dimen/htc_man_stateMessage2_layout_margin_left" />
-                <ImageView android:layout_gravity="right|center" android:id="@id/icon_storage_indicate" android:visibility="invisible" android:layout_width="wrap_content" android:layout_height="wrap_content" android:layout_marginLeft="@dimen/htc_man_icon_storage_indicate_layout_margin_left" android:layout_alignParentRight="true" />
-                <TextView android:ellipsize="end" android:gravity="right|center" android:id="@id/stateFileIndex" android:layout_width="@dimen/htc_man_state_file_index_layout_width" android:layout_height="wrap_content" android:text="@string/default_file_pos_of_all" android:singleLine="true" android:layout_toLeftOf="@id/icon_storage_indicate" />
-            </RelativeLayout>
-            <com.htc.soundrecorder.VUMeter android:layout_gravity="center_horizontal" android:id="@id/uvMeter" android:layout_width="@dimen/htc_man_uvmeter_layout_width" android:layout_height="@dimen/htc_man_uvmeter_layout_height" android:layout_marginLeft="@dimen/htc_man_uvmeter_layout_margin_left" android:layout_below="@id/stateMessage2Layout" />
-            <RelativeLayout android:id="@id/timerViewLayout" android:layout_width="@dimen/htc_man_timer_viewlayout_layout_width" android:layout_height="@dimen/htc_man_timer_Viewlayout_layout_height" android:layout_below="@id/uvMeter" android:layout_centerHorizontal="true">
-                <com.htc.soundrecorder.TimeView android:id="@id/timerView" android:layout_width="fill_parent" android:layout_height="wrap_content" android:layout_centerHorizontal="true" style="@android:style/TextAppearance.Large" />
+    <RelativeLayout android:orientation="horizontal" android:id="@id/effectViewLayout0" android:layout_width="wrap_content" android:layout_height="wrap_content" android:layout_marginLeft="6.0px" android:layout_marginTop="0.0px">
+        <RelativeLayout android:orientation="horizontal" android:id="@id/effectViewLayoutBackground" android:background="@drawable/screen_on" android:layout_width="228.0px" android:layout_height="145.0px" />
+        <RelativeLayout android:id="@id/effectViewLayout" android:layout_width="228.0px" android:layout_height="145.0px">
+            <RelativeLayout android:orientation="horizontal" android:id="@id/stateMessage2Layout" android:layout_width="fill_parent" android:layout_height="20.0px" android:layout_marginTop="5.0px" android:baselineAligned="false" android:layout_centerHorizontal="true" style="@style/voiceRecordTextStytleB">
+                <TextView android:textSize="12.0px" android:layout_gravity="left|center" android:id="@id/stateMessage2" android:layout_width="140.0px" android:layout_height="wrap_content" android:layout_marginLeft="25.0px" />
+                <ImageView android:layout_gravity="right|center" android:id="@id/icon_storage_indicate" android:visibility="invisible" android:layout_width="wrap_content" android:layout_height="wrap_content" android:layout_marginLeft="8.0px" android:layout_alignParentRight="true" />
+                <TextView android:textSize="12.0px" android:ellipsize="end" android:gravity="right|center" android:id="@id/stateFileIndex" android:layout_width="100.0px" android:layout_height="wrap_content" android:text="@string/default_file_pos_of_all" android:singleLine="true" android:layout_toLeftOf="@id/icon_storage_indicate" />
             </RelativeLayout>
+            <com.htc.soundrecorder.VUMeter android:layout_gravity="center_horizontal" android:id="@id/uvMeter" android:layout_width="184.0px" android:layout_height="100.0px" android:layout_marginLeft="33.0px" android:layout_marginTop="-20.0px" android:layout_below="@id/stateMessage2Layout" />
+            <com.htc.soundrecorder.TimeView android:textSize="50.0px" android:id="@id/timerView" android:layout_width="fill_parent" android:layout_height="100.0px" android:layout_marginLeft="5.0px" android:layout_marginTop="85.0px" style="@android:style/TextAppearance.Large" />
         </RelativeLayout>
-        <RelativeLayout android:orientation="vertical" android:id="@id/playViewLayout" android:background="@drawable/screen_playback" android:visibility="gone" android:layout_width="@dimen/htc_man_play_viewlayout_layout_width" android:layout_height="@dimen/htc_man_play_viewlayout_layout_height">
-            <TextView android:ellipsize="end" android:gravity="center_horizontal" android:id="@id/playstaus" android:visibility="invisible" android:layout_width="fill_parent" android:layout_height="wrap_content" android:layout_marginTop="@dimen/htc_man_playstaus_margin_top" android:text="@string/htc_playing" android:singleLine="true" style="@style/voiceRecordTextStytleB" />
-            <RelativeLayout android:id="@id/seekbarlayout" android:layout_width="fill_parent" android:layout_height="@dimen/htc_man_seekbarlayout_layout_height" android:layout_below="@id/playstaus">
+        <RelativeLayout android:orientation="vertical" android:id="@id/playViewLayout" android:background="@drawable/screen_playback" android:visibility="gone" android:layout_width="228.0px" android:layout_height="145.0px">
+            <TextView android:textSize="22.0px" android:ellipsize="end" android:gravity="center_horizontal" android:id="@id/playstaus" android:visibility="invisible" android:layout_width="fill_parent" android:layout_height="wrap_content" android:layout_marginTop="@dimen/htc_man_playstaus_margin_top" android:text="@string/htc_playing" android:singleLine="true" style="@style/voiceRecordTextStytleB" />
+            <RelativeLayout android:id="@id/seekbarlayout" android:layout_width="220.0px" android:layout_height="@dimen/htc_man_seekbarlayout_layout_height" android:layout_marginBottom="5.0px" android:layout_below="@id/playstaus">
                 <include layout="@layout/main_seekbar" />
             </RelativeLayout>
             <RelativeLayout android:id="@id/palypostextlayout" android:layout_width="fill_parent" android:layout_height="@dimen/htc_man_palypostextlayout_layout_height" android:layout_below="@id/seekbarlayout">
-                <TextView android:ellipsize="end" android:gravity="left|center" android:id="@id/cur_pos_l" android:layout_width="@dimen/htc_man_cur_pos_l_layout_width" android:layout_height="wrap_content" android:layout_marginLeft="@dimen/htc_man_cur_pos_l_layout_margin_left" android:text="@string/default_file_timepos" android:singleLine="true" style="@style/voiceRecordTextStytleC" />
-                <TextView android:ellipsize="end" android:gravity="right|center" android:id="@id/dur_remain_r" android:layout_width="@dimen/htc_man_dur_remain_r_layout_width" android:layout_height="wrap_content" android:layout_marginRight="@dimen/htc_man_dur_remain_r_layout_margin_right" android:text="@string/default_file_timepos" android:singleLine="true" android:layout_alignParentRight="true" style="@style/voiceRecordTextStytleC" />
+                <TextView android:ellipsize="end" android:gravity="left|center" android:id="@id/cur_pos_l" android:layout_width="40.0px" android:layout_height="wrap_content" android:layout_marginLeft="8.0px" android:text="@string/default_file_timepos" android:singleLine="true" style="@style/voiceRecordTextStytleC" />
+                <TextView android:ellipsize="end" android:gravity="right|center" android:id="@id/dur_remain_r" android:layout_width="40.0px" android:layout_height="wrap_content" android:layout_marginRight="8.0px" android:text="@string/default_file_timepos" android:singleLine="true" android:layout_alignParentRight="true" style="@style/voiceRecordTextStytleC" />
             </RelativeLayout>
         </RelativeLayout>
     </RelativeLayout>
diff -rupN ../original-apks/HtcSoundRecorder.apk/res/layout-mdpi/main_seekbar.xml HtcSoundRecorder.apk/res/layout-mdpi/main_seekbar.xml
--- ../original-apks/HtcSoundRecorder.apk/res/layout-mdpi/main_seekbar.xml	2012-05-26 10:18:01.000000000 +0100
+++ HtcSoundRecorder.apk/res/layout-mdpi/main_seekbar.xml	2012-05-26 10:08:53.000000000 +0100
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <merge
   xmlns:android="http://schemas.android.com/apk/res/android">
-    <com.htc.soundrecorder.util.HtcSeekBar android:gravity="center" android:id="@id/stateProgressBar" android:layout_width="233.0px" android:layout_height="22.0px" android:layout_marginTop="2.0px" android:layout_centerHorizontal="true" app:max="100" app:progress="0" app:progressDrawable="@drawable/progress_horizontal_pressed" app:minHeight="14.0px" app:maxHeight="14.0px" app:thumb="@com.htc:drawable/common_music_progress_button" app:thumbOffset="0.0px" app:pressed_progressDrawable="@drawable/progress_horizontal_pressed" app:pressed_minHeight="14.0px" app:pressed_maxHeight="14.0px" app:pressed_thumb="@com.htc:drawable/common_music_progress_button" app:pressed_thumbOffset="@dimen/seekbar_pressed_thumboffset"
+    <com.htc.soundrecorder.util.HtcSeekBar android:gravity="center" android:id="@id/stateProgressBar" android:layout_width="210.0px" android:layout_height="40.0dip" android:layout_marginTop="0.0px" android:layout_centerHorizontal="true" app:max="100" app:progress="0" app:progressDrawable="@drawable/progress_horizontal_pressed" app:minHeight="40.0px" app:maxHeight="40.0px" app:thumb="@com.htc:drawable/common_music_progress_button" app:thumbOffset="8.0px" app:pressed_progressDrawable="@drawable/progress_horizontal_pressed" app:pressed_minHeight="40.0px" app:pressed_maxHeight="40.0px" app:pressed_thumb="@com.htc:drawable/common_music_progress_button" app:pressed_thumbOffset="@dimen/seekbar_pressed_thumboffset" app:progress_paddingTop="10.0px" app:pressed_paddingTop="10.0px"
       xmlns:app="http://schemas.android.com/apk/res/com.htc.soundrecorder" />
 </merge>
diff -rupN ../original-apks/HtcSoundRecorder.apk/res/layout-mdpi/main.xml HtcSoundRecorder.apk/res/layout-mdpi/main.xml
--- ../original-apks/HtcSoundRecorder.apk/res/layout-mdpi/main.xml	2012-05-26 10:18:01.000000000 +0100
+++ HtcSoundRecorder.apk/res/layout-mdpi/main.xml	2012-05-26 10:08:53.000000000 +0100
@@ -1,16 +1,16 @@
 <?xml version="1.0" encoding="utf-8"?>
-<LinearLayout android:orientation="vertical" android:background="@drawable/screen_bg" android:layout_width="fill_parent" android:layout_height="fill_parent"
+<LinearLayout android:orientation="vertical" android:background="@drawable/screen_bg" android:layout_width="fill_parent" android:layout_height="295.0px"
   xmlns:android="http://schemas.android.com/apk/res/android">
     <LinearLayout android:orientation="vertical" android:id="@id/man_upper" android:layout_width="fill_parent" android:layout_height="@dimen/htc_man_upper_layout_height">
         <include layout="@layout/main_titlebar" />
         <include layout="@layout/effect_panel" />
         <RelativeLayout android:layout_width="fill_parent" android:layout_height="fill_parent">
-            <TextView android:ellipsize="marquee" android:gravity="center" android:id="@id/fileName" android:layout_width="fill_parent" android:layout_height="@dimen/htc_man_filename_layout_height" android:layout_marginTop="@dimen/htc_man_filename_layout_margin_top" android:singleLine="true" android:marqueeRepeatLimit="marquee_forever" style="@style/voiceRecordTextStytleB" />
-            <TextView android:gravity="center" android:id="@id/stateMessage1" android:paddingLeft="20.0px" android:paddingRight="20.0px" android:layout_width="wrap_content" android:layout_height="wrap_content" android:layout_marginTop="@dimen/htc_man_state_message1_margin_top" android:layout_centerHorizontal="true" style="@style/voiceRecordTextStytleC" />
+            <TextView android:ellipsize="marquee" android:gravity="center" android:id="@id/fileName" android:layout_width="fill_parent" android:layout_height="wrap_content" android:layout_marginTop="30.0px" android:singleLine="true" android:marqueeRepeatLimit="marquee_forever" style="@style/voiceRecordTextStytleB" />
+            <TextView android:gravity="center" android:id="@id/stateMessage1" android:layout_width="wrap_content" android:layout_height="wrap_content" android:layout_marginTop="5.0px" android:singleLine="true" android:layout_centerHorizontal="true" style="@style/voiceRecordTextStytleC" />
             <TextView android:gravity="center" android:id="@id/stateMessage_error" android:padding="@dimen/htc_man_state_message_erro_padding" android:layout_width="wrap_content" android:layout_height="wrap_content" android:layout_centerInParent="true" style="@style/voiceRecordTextStytleC" />
         </RelativeLayout>
     </LinearLayout>
-    <LinearLayout android:layout_width="wrap_content" android:layout_height="wrap_content" android:layout_marginTop="@dimen/htc_man_taskbar_margin_top">
+    <LinearLayout android:layout_width="wrap_content" android:layout_height="40.0px" android:layout_marginTop="@dimen/htc_man_taskbar_margin_top">
         <include layout="@com.htc:layout/taskbar_2" />
     </LinearLayout>
 </LinearLayout>
diff -rupN ../original-apks/HtcSoundRecorder.apk/res/layout-mdpi/statusbar.xml HtcSoundRecorder.apk/res/layout-mdpi/statusbar.xml
--- ../original-apks/HtcSoundRecorder.apk/res/layout-mdpi/statusbar.xml	2012-05-26 10:18:01.000000000 +0100
+++ HtcSoundRecorder.apk/res/layout-mdpi/statusbar.xml	2012-05-26 10:08:53.000000000 +0100
@@ -3,10 +3,10 @@
   xmlns:android="http://schemas.android.com/apk/res/android">
     <LinearLayout android:gravity="center_vertical" android:layout_width="fill_parent" android:layout_height="wrap_content">
         <ImageView android:gravity="center" android:id="@id/icon" android:padding="4.0dip" android:layout_width="32.0dip" android:layout_height="32.0dip" android:layout_marginLeft="2.0dip" android:layout_marginTop="4.0dip" />
-        <TextView android:textAppearance="?android:textAppearanceMediumInverse" android:ellipsize="marquee" android:id="@id/appname" android:focusable="true" android:layout_width="wrap_content" android:layout_height="wrap_content" android:layout_marginLeft="2.0dip" android:layout_marginTop="4.0dip" android:text="@string/htc_app_name" android:singleLine="true" android:layout_alignParentLeft="true" />
+        <TextView android:textAppearance="?android:textAppearanceMediumInverse" android:textSize="@dimen/text_size_primary" android:textStyle="bold" android:ellipsize="marquee" android:id="@id/appname" android:focusable="true" android:layout_width="wrap_content" android:layout_height="wrap_content" android:layout_marginLeft="2.0dip" android:layout_marginTop="4.0dip" android:text="@string/htc_app_name" android:singleLine="true" android:layout_alignParentLeft="true" />
     </LinearLayout>
     <LinearLayout android:paddingTop="1.0dip" android:layout_width="fill_parent" android:layout_height="wrap_content">
-        <TextView android:textAppearance="?android:textAppearanceSmallInverse" android:id="@id/status" android:paddingLeft="4.0dip" android:layout_width="wrap_content" android:layout_height="wrap_content" android:layout_marginLeft="8.0dip" android:layout_marginTop="2.0dip" android:text="@string/recording" android:layout_alignParentLeft="true" />
-        <TextView android:textAppearance="?android:textAppearanceSmallInverse" android:id="@id/progress" android:paddingRight="4.0dip" android:layout_width="wrap_content" android:layout_height="wrap_content" android:layout_alignParentRight="true" />
+        <TextView android:textAppearance="?android:textAppearanceSmallInverse" android:textSize="@dimen/text_size_secondary" android:id="@id/status" android:paddingLeft="4.0dip" android:layout_width="wrap_content" android:layout_height="wrap_content" android:layout_marginLeft="5.0dip" android:layout_marginTop="2.0dip" android:text="@string/recording" android:layout_alignParentLeft="true" />
+        <TextView android:textAppearance="?android:textAppearanceSmallInverse" android:textSize="@dimen/text_size_secondary" android:id="@id/progress" android:paddingRight="4.0dip" android:layout_width="wrap_content" android:layout_height="wrap_content" android:layout_alignParentRight="true" />
     </LinearLayout>
 </LinearLayout>
diff -rupN ../original-apks/HtcSoundRecorder.apk/res/values-mdpi/dimens.xml HtcSoundRecorder.apk/res/values-mdpi/dimens.xml
--- ../original-apks/HtcSoundRecorder.apk/res/values-mdpi/dimens.xml	2012-05-26 10:18:01.000000000 +0100
+++ HtcSoundRecorder.apk/res/values-mdpi/dimens.xml	2012-05-26 10:08:53.000000000 +0100
@@ -1,20 +1,20 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <dimen name="seekbar_pressed_thumboffset">0.0px</dimen>
-    <dimen name="htc_man_taskbar_margin_top">4.0px</dimen>
-    <dimen name="htc_main_title_bar_layout1_layout_height">31.0px</dimen>
-    <dimen name="htc_main_title_bar_layout1_layout_margin_top">3.0px</dimen>
-    <dimen name="htc_man_activity_title_shadow_layout_height">29.0px</dimen>
+    <dimen name="seekbar_pressed_thumboffset">8.0px</dimen>
+    <dimen name="htc_man_taskbar_margin_top">3.0px</dimen>
+    <dimen name="htc_main_title_bar_layout1_layout_height">28.0px</dimen>
+    <dimen name="htc_main_title_bar_layout1_layout_margin_top">0.0px</dimen>
+    <dimen name="htc_man_activity_title_shadow_layout_height">20.0px</dimen>
     <dimen name="htc_man_activity_title_shadow_max_width">310.0px</dimen>
-    <dimen name="htc_man_activity_title_shadow_layout_margin_top">2.0px</dimen>
+    <dimen name="htc_man_activity_title_shadow_layout_margin_top">4.0px</dimen>
     <dimen name="htc_main_activity_title_layout_margin_left">1.0px</dimen>
-    <dimen name="htc_main_activity_title_layout_margin_top">3.0px</dimen>
-    <dimen name="htc_man_upper_layout_height">395.0px</dimen>
+    <dimen name="htc_main_activity_title_layout_margin_top">5.0px</dimen>
+    <dimen name="htc_man_upper_layout_height">255.0px</dimen>
     <dimen name="htc_man_filename_layout_margin_top">58.0px</dimen>
     <dimen name="htc_man_filename_layout_height">35.0px</dimen>
     <dimen name="htc_man_state_message1_margin_top">98.0px</dimen>
     <dimen name="htc_man_state_message_erro_padding">20.0px</dimen>
-    <dimen name="htc_man_effectViewlayout0_layout_margin_top">7.0px</dimen>
+    <dimen name="htc_man_effectViewlayout0_layout_margin_top">11.0px</dimen>
     <dimen name="htc_man_effectViewlayout0_layout_margin_left">9.0px</dimen>
     <dimen name="htc_man_effectViewlayout_layout_width">320.0px</dimen>
     <dimen name="htc_man_effectViewlayout_layout_height">192.0px</dimen>
@@ -38,15 +38,15 @@
     <dimen name="htc_man_cur_pos_l_layout_margin_left">35.0px</dimen>
     <dimen name="htc_man_dur_remain_r_layout_width">80.0dip</dimen>
     <dimen name="htc_man_dur_remain_r_layout_margin_right">37.0px</dimen>
-    <dimen name="htc_man_state_message_padding_left">8.0px</dimen>
-    <dimen name="htc_man_state_message_padding_right">8.0px</dimen>
-    <dimen name="htc_list_item_margin_top">3.0px</dimen>
-    <dimen name="htc_list_item_margin_left">8.0px</dimen>
+    <dimen name="htc_man_state_message_padding_left">6.0px</dimen>
+    <dimen name="htc_man_state_message_padding_right">6.0px</dimen>
+    <dimen name="htc_list_item_margin_top">4.0px</dimen>
+    <dimen name="htc_list_item_margin_left">6.0px</dimen>
     <dimen name="htc_list_item_margin_right">5.0px</dimen>
     <dimen name="htc_list_item_height">50.0px</dimen>
-    <dimen name="htc_list_item_margin_left_add_text">5.0px</dimen>
-    <dimen name="text_size_primary">18.0px</dimen>
-    <dimen name="text_size_secondary">14.0px</dimen>
+    <dimen name="htc_list_item_margin_left_add_text">6.0px</dimen>
+    <dimen name="text_size_primary">16.0px</dimen>
+    <dimen name="text_size_secondary">13.0px</dimen>
     <dimen name="htc_main_text_stytle_a_layout_height">32.0px</dimen>
-    <dimen name="htc_main_text_stytle_a_text_size">18.0px</dimen>
+    <dimen name="htc_main_text_stytle_a_text_size">14.0px</dimen>
 </resources>
diff -rupN ../original-apks/HtcSoundRecorder.apk/res/values-mdpi/styles.xml HtcSoundRecorder.apk/res/values-mdpi/styles.xml
--- ../original-apks/HtcSoundRecorder.apk/res/values-mdpi/styles.xml	2012-05-26 10:18:01.000000000 +0100
+++ HtcSoundRecorder.apk/res/values-mdpi/styles.xml	2012-05-26 10:08:53.000000000 +0100
@@ -23,12 +23,12 @@
         <item name="android:layout_marginBottom">2.0px</item>
     </style>
     <style name="voiceRecordTextStytleB">
-        <item name="android:textSize">22.0px</item>
+        <item name="android:textSize">16.0px</item>
         <item name="android:textColor">@color/text_color</item>
         <item name="android:gravity">center</item>
     </style>
     <style name="voiceRecordTextStytleC">
-        <item name="android:textSize">16.0px</item>
+        <item name="android:textSize">12.0px</item>
         <item name="android:textColor">@color/text_clor_minor</item>
         <item name="android:layout_gravity">center</item>
         <item name="android:layout_width">fill_parent</item>
@@ -39,7 +39,7 @@
         <item name="android:shadowRadius">1.0</item>
     </style>
     <style name="stylecheckBoxText">
-        <item name="android:textSize">18.0px</item>
+        <item name="android:textSize">13.0px</item>
         <item name="android:textColor">@color/text_color</item>
         <item name="android:gravity">center</item>
     </style>
